### PR TITLE
fix: log printing and decoding

### PR DESCRIPTION
**Pull request by banteg**
- fix: event name
- fix: insufficient data bytes

### What I did

fixes: #

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: src/ape_ethereum/ecosystem.py [hunk 1]
```diff
@@ -464,7 +464,10 @@ def decode_logs(self, logs: List[Dict], *events: EventABI) -> Iterator["Contract
             except KeyError:
                 continue
 
-            event_arguments = abi.decode(topics, log["data"])
+            try:
+                event_arguments = abi.decode(topics, log["data"])
+            except InsufficientDataBytes:
+                continue
```
Comment (fubuloubu) [inline src/ape_ethereum/ecosystem.py:470]: Would love a debug log here

**Review by fubuloubu — APPROVED** [review]

File: src/ape_ethereum/ecosystem.py [hunk 2]
```diff
@@ -464,7 +465,11 @@ def decode_logs(self, logs: List[Dict], *events: EventABI) -> Iterator["Contract
             except KeyError:
                 continue
 
-            event_arguments = abi.decode(topics, log["data"])
+            try:
+                event_arguments = abi.decode(topics, log["data"])
+            except InsufficientDataBytes:
+                logger.debug("failed to decode log data for %s", log)
```
Comment (fubuloubu) [inline src/ape_ethereum/ecosystem.py:471]:
```suggestion
                logger.debug(f"failed to decode log data for {log}")
```

**Review by fubuloubu — DISMISSED** [review]

File: src/ape_ethereum/ecosystem.py (hunk 2 above)
Comment (banteg) [reply src/ape_ethereum/ecosystem.py]: https://www.palkeo.com/en/blog/python-logging.html

File: src/ape_ethereum/ecosystem.py [hunk 3]
```diff
@@ -464,7 +465,11 @@ def decode_logs(self, logs: List[Dict], *events: EventABI) -> Iterator["Contract
             except KeyError:
                 continue
 
-            event_arguments = abi.decode(topics, log["data"])
+            try:
+                event_arguments = abi.decode(topics, log["data"])
+            except InsufficientDataBytes:
+                logger.debug(f"failed to decode log data for {log}")
```
Comment (fubuloubu) [inline src/ape_ethereum/ecosystem.py:471]:
```suggestion
                logger.debug("failed to decode log data for %s", log, exc_info=True)
```

**Review by fubuloubu — APPROVED** [review]

File: src/ape/types/__init__.py [hunk 4]
```diff
@@ -190,7 +190,7 @@ def validate_address(cls, value):
 
     def __str__(self) -> str:
         args = " ".join(f"{key}={val}" for key, val in self.event_arguments.items())
-        return f"{self.name} {args}"
+        return f"{self.event_name} {args}"
```
Comment (unparalleled-js) [inline src/ape/types/__init__.py:193]: wouldn't mind a `test_repr` to ensure this doesn't happen again

**Pull request merged**